### DISCUSSION TITLE
Unit change in export command.

### DISCUSSION THREAD

**Discussion by engphb** [PMEAL/OpenPNM#2654] [Q&A]
@jgostick. Dear professor, I need to export my network to view it in Paraview, but by default, the network is in meters, in Paraview I would like to view the properties in micrometers. The export command I'm using is the one below. Is there any way to perform this transformation of units in the export command?

op.io.project_to_vtk(proj_1, filename='network', fill_nans=None, fill_infs=None)

**Comment by jgostick** [comment]
There is not way to do this in OpenPNM, but am I correct to assume that you are trying to overlay a network and a voxel image?  If so then you can adjust the scaling of the voxel image in the property pipeline in paraview.

**Reply by engphb** [reply]
Hello professor, I don't know how to adjust the scaling of the voxel image in the property pipeline in ParaView. Is there any tutorial that teaches how to do this?

**Reply by jgostick** [reply]
There is a checkbox called "Use custom data spacing" where you put the voxel lengths, in m so it matches openpnm

**Reply by engphb** [reply]
Hello professor, I couldn't find this checkbox called "Use custom data spacing". Is this name correct?

**Reply by jgostick** [reply]
![Picture1](https://user-images.githubusercontent.com/5092049/209983934-92df5cbd-804d-40a8-bac8-169ac41b1a8d.png)

**Reply by engphb** [reply]
Thank you, professor! I noticed from your figure that the file must be in .tif for this option to appear. I was working on .vti.
